qa/TL0_python-self-test/test.sh

```diff
@@ -1,2 +1,9 @@
 #!/bin/bash -e
-./test_nofw.sh
\ No newline at end of file
+
```

Comment:
Do we still need need `TL0_python-self-test`?

Reply:
So my idea is to keep it and run all the split test in there, otherwise we would have new jobs: 3 * `number of supported architectures` in the CI. And this way we can run it in one go when doing a build with L1.

Reply:
So please remove other files from qa/TL0_python-self-test, like `test_body.sh`, `test_no_setup.sh` or `test_nofw.sh`.

Reply:
I already did.

Reply:
Ok. Git shows them as moved. Missed that.